app/views/users/service_provider_inactive/index.html.erb

```diff
@@ -1,21 +1,16 @@
 <% self.title = t('service_providers.errors.inactive.heading', sp_name: @sp_name, app_name: APP_NAME) %>
 
-<%= render AlertIconComponent.new(icon_name: :error) %>
+<%= render StatusPageComponent.new(status: :error) do |c| %>
+  <% c.with_header { t('service_providers.errors.inactive.heading', sp_name: @sp_name, app_name: APP_NAME) } %>
 
-<h3 class="margin-bottom-1 margin-top-4">
-  <%= t('service_providers.errors.inactive.heading', sp_name: @sp_name, app_name: APP_NAME) %>
-</h3>
+  <p>
+    <%= t('service_providers.errors.inactive.instructions', sp_name: @sp_name, app_name: APP_NAME) %>
+  </p>
 
-<p>
-  <%= t('service_providers.errors.inactive.instructions', sp_name: @sp_name, app_name: APP_NAME) %>
-</p>
+  <p>
+    <%= t('service_providers.errors.inactive.instructions2') %>
+  </p>
 
-<p>
-  <%= t('service_providers.errors.inactive.instructions2', sp_link: @sp_link) %>
```

Comment:
The `instructions2` text doesn't actually interpolate an `sp_link` variable, and it's not assigned in the controller.

https://github.com/18F/identity-idp/blob/20fe4239526d24ae0085b40cc4fe9699b45e5d3b/config/locales/en.yml#L1478

It might be nice to have some sort of linter which checks that we don't call `t` with useless variables.